core/src/main/java/com/amplifyframework/api/graphql/QueryType.java

```diff
@@ -27,7 +27,12 @@
     /**
      * GraphQL query list.
      */
-    LIST;
+    LIST,
+
+    /**
+     * GraphQL query sync.
+     */
+    SYNC;
```

Comment:
Almost certainly not a comment for this PR, but while I'm here:

I hadn't thought about this at the time, but _get_, _list_, _sync_: these are really verbs from the AppSync domain, aren't they? GraphQL itself has no such pre-defined operation types. Should we aim to move the `QueryType` enum into `aws-api-appsync`?

Reply:
That makes sense, I will move it to `aws-api-appsync`.  I think there's several more things like this that I will also be moving over soon.